fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -3328,24 +3328,43 @@ void DDTeamCollection::updateCpuPivots() {
 }
 
 void DDTeamCollection::updateTeamEligibility() {
+	int healthyCount = 0, lowDiskUtilTotal = 0, lowCpuTotal = 0, lowDiskUtilLowCpu = 0;
 	for (auto& team : teams) {
 		if (team->isHealthy()) {
+			bool lowDiskUtil = false, lowCPU = false;
```

Comment:
Nit: 
```suggestion
			bool lowDiskUtil = team->hasHealthyAvailableSpace(teamPivots.pivotAvailableSpaceRatio), lowCPU = team->hasLowerCpu(teamPivots.pivotCPU);
```
Less assignment operation.

Reply:
If we are doing this, we would call `hasHealthyAvailableSpace` and `hasLowerCpu` two times in this function, which can add some overhead as well?